src/util/diffsHandler.js

```diff
@@ -12,14 +12,15 @@ require('dotenv').config();
 * aka_all_data - object that contain all the recent data from aka
 */
 
-module.exports = async ({ added = [], updated = [] }, dataSource, aka_all_data, runnigType, sendLog, Auth) => {
+module.exports = async ({ added = [], updated = [] }, dataSource, aka_all_data, ct_all_data, pictures_all_data, runnigType, sendLog, Auth) => {
```

Comment:
1. merge aka_all_data and ct_all_data into extraData
2. pictures_all_data goes into aka_all_data, and taking care of in completefromAka